gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 120
+cordaApiRevision = 121
```

Comment:
Beware there is already a 121 approved: https://github.com/corda/corda-api/pull/425
and will be merged when @JamesHR3's duplicate 120 is backed out, so probably best to synch up with @JamesHR3 in terms of who merges first.

Reply:
121 is merged and build is back green: https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%2Fcorda-api/detail/release%2Fos%2F5.0/766/pipeline/104